src/components/Form/index.tsx

```diff
@@ -0,0 +1,86 @@
+import RjsForm from '@rjsf/mui';
+import ajvValidator from '@rjsf/validator-ajv8';
+import { FormProps as RjsFormProps } from '@rjsf/core';
+import { useCallback } from 'react';
+import { Box, Button, ButtonProps } from '@mui/material';
+import { PasswordWidget } from './PasswordWidget';
+import { TextWidget } from './TextWidget';
+import { TextareaWidget } from './TextareaWidget';
+import { SelectWidget } from './SelectWidget';
+import { DateTimeWidget } from './DateTimeWidget';
+import { UpDownWidget } from './UpDownWidget';
+
+const internalWidgets: {
+	[k: string]: any;
+} = {
+	PasswordWidget,
+	EmailWidget: TextWidget,
+	TextWidget,
+	TextareaWidget,
+	SelectWidget,
+	UpDownWidget,
+	DateTimeWidget,
+};
+
+export interface MuiFormProps extends Omit<RjsFormProps, 'validator'> {
+	/** If true, do not display the form submit button */
+	hideSubmitButton?: boolean;
+	/** Properties that are passed to the submit button, these are the same props used for the [`Button`](#button) component */
+	submitButtonProps?: ButtonProps;
+	/** If passed, it will show a secondary button, these are the same props used for the [`Button`](#button) component */
+	secondaryButtonProps?: ButtonProps;
+
+	validator?: RjsFormProps['validator'];
+}
+
+export const MuiForm: React.FC<React.PropsWithChildren<MuiFormProps>> = ({
```

Comment:
Why name it `MuiForm` instead of just `Form`? I would prefer `RJSForm`, especially in the unlikely/faraway scenario that we switch from MUI

Reply:
I would not use Form as MUI might create a Form component at some point which would create confusion (no conflict but still confusing IMHO). I can go with RJSF but seems strange as it would be react-json-schema-form-form

Reply:
`RJSForm` :)